docs/code_tour.md

```diff
@@ -2,8 +2,37 @@
 This document is intended to provide a development guideline for new and existing contributors to the project.
 Keep in mind this application undergoes constant development so be sure to check back occasionally as best practices are sure to evolve.
 
-## Linting
-This project uses **ESLint** and **Sass Lint**. Both can be run with the script `yarn vet`.
+## Yarn Tasks 
+#### `yarn build`
+- Bundles SUI assets and places then in `../manageiq/public/ui/service/`.
```

Comment:
It may be differences in markdown formatters, but with GH-flavored markdown (using [octodown](https://github.com/ianks/octodown) gem) it isn't being rendered properly.

<img width="968" alt="screen shot 2017-04-26 at 1 51 37 pm" src="https://cloud.githubusercontent.com/assets/6842753/25448752/822bcc68-2a87-11e7-906e-41f003f320a5.png">

Looks like [this plugin](https://plugins.jetbrains.com/plugin/7701-gfm) works with RubyMine to preview exactly what `.md` files will look like on GH.